lib/friends/introvert.rb

```diff
@@ -658,6 +658,19 @@ def set_n_activities!(type)
       end
     end
 
+    def set_implicit_locations(_activities)
+      implicit_location = nil
```

Comment:
I _think_ this line is unnecessary, but I could be getting my scoping rules mixed up.

Reply:
No you are correct, I'll remove this 👍

Reply:
No you are correct, I will remove

Reply:
It's needed at this level to be available at different scopes (removing it causes tests to fail), so I'll keep it in.